src/datasources/customer.datasource.ts

```diff
@@ -4,21 +4,18 @@ import {juggler} from '@loopback/repository';
 const config = {
   name: 'Customer',
   connector: 'postgresql',
-  url: 'postgres://username:password@localhost:5454/sourcefuse-training',
-  host: 'localhost',
-  port: 5454,
-  user: 'username',
-  password: 'password',
```

Comment:
these configs are needed for DB connection 
they should be passed as env

Reply:
Yes, those are deleted lines, it's coming from env files only.

[Please check here](https://github.com/AryanshMahato/loopback-training/blob/middleware/src/datasources/customer.datasource.ts#L7)